src/job_svc/mod.rs

```diff
@@ -0,0 +1,16 @@
+mod populate_outbox;
+
+use job_crate::Jobs;
+
+use crate::job_svc::populate_outbox::PopulateOutboxJobInit;
+use crate::{ledger::Ledger, outbox::Outbox};
+
+pub use populate_outbox::spawn_outbox_handler;
+
+pub struct JobSvc;
+
+impl JobSvc {
+    pub fn init(jobs: &Jobs, outbox: Outbox, ledger: Ledger) {
```

Comment:
`init` -> async, can fail creation pattern
`pub async fn init() -> Result<Self, Error>`

if creation is not async (and therefore can never fail)

`pub new() -> Self`